### ISSUE TITLE
Warning :131] Could not set performance expectation : 52 Inappropriate ioctl for device

### ISSUE THREAD

**Issue by Pete1979** [blakeblackshear/frigate#1788]
Hi, 
I just installed a PCIe Coral in my PC with Home Assistant OS 6.4.
Changed the configuration in frigate.yml to Coral detector,
Frigate starts fine and seams to be working as it should.
But i'm getting a warning message in the log as follows:
W :131] Could not set performance expectation : 52 (Inappropriate ioctl for device)

**Comment by splaspood** [comment]
I don't believe I encountered this specific message but I had similar issues about inappropriate ioctl before I compiled the vendor-provided driver for the coral.   https://coral.ai/docs/m2/get-started/

**Comment by Pete1979** [comment]
Thing is, since I'm running Home assistant OS, I don't have proper access to ssh terminal. (need to do something with a USB stick and Auth keys)
And from HAOS version 6, drivers should be included. And Frigate handles it. 
Also wondering if the warning means that something isent working as it should.
